packages/server/tests/acceptance/conformityTests.spec.ts

```diff
@@ -204,6 +208,34 @@ function checkResponseFormat(fileName, actualReponse, expectedResponse) {
   return missingKeys;
 }
 
+function findSchema(file) {
+  const execApisOpenRpcData = require('../../../../openrpc_exec_apis.json');
+  const schema = execApisOpenRpcData.methods.find((method) => method.name === file)?.result?.schema;
+
+  return schema;
+}
+
+function isResponseValid(schema, response) {
+  const validate = ajv.compile(schema);
+
+  const valid = validate(response.result);
+  expect(validate.errors).to.be.null;
+
+  return valid;
+}
+
+function validateBlockNumber() {
+  const schema = {
+    title: 'hex encoded unsigned integer',
+    type: 'string',
+    pattern: '^0x([1-9a-f]+[0-9a-f]*|0)$',
+  };
+  const validate = ajv.compile(schema);
+  const valid = validate('d76');
```

Comment:
What are we validating here exactly? It seems like this is a hardcoded string.

Reply:
Leftovers from testing that I will remove, thanks for pointing out